### DISCUSSION TITLE
Running multiple R cells simultaneously

### DISCUSSION THREAD

**Discussion by pehkawn** [microsoft/vscode-jupyter#10215] [General]
I am working on a rather computationally heavy model, which I am building step-by-step. Since it takes some time to compute parts of the model, it would be prudent to be able to work on the next steps of the model while the computations are running. This, naturally, entails a bit of testing of code.
Therefore, I was wondering if there's a way to execute run in multiple cells simultaneously? (The Jupyter notebook already seems to start multiple kernels, while most of the seem to remain inactive..)

**Comment by rchiodo** [comment]
I don't believe this is possible for an 'R' kernel. However IPython does have something that sort of supports it. See this here:
https://stackoverflow.com/questions/21248345/is-there-a-way-to-run-multiple-cells-simultaneously-in-ipython-notebook
